monitored-httpclient/src/main/java/com/github/nhenneaux/resilienthttpclient/monitoredclientpool/HttpClientPool.java

```diff
@@ -49,7 +50,7 @@ public HttpClientPool(
             final ServerConfiguration serverConfiguration,
             final KeyStore trustStore
     ) {
-        this(dnsLookupWrapper, scheduledExecutorService, serverConfiguration, trustStore, HttpClient.newBuilder());
+        this(dnsLookupWrapper, scheduledExecutorService, serverConfiguration, trustStore, HttpClient.newBuilder().connectTimeout(Duration.ofSeconds(2)));
```

Comment:
Not configurable? :(

Reply:
it is using the right constructor taking a client builder as parameter. https://github.com/nhenneaux/resilient-httpclient/blob/master/monitored-httpclient/src/main/java/com/github/nhenneaux/resilienthttpclient/monitoredclientpool/HttpClientPool.java#L55

Having an infinite connect timeout is not a relevant configuration especially when you check the availability of sockets.